bindings/c/test/mako/mako.cpp

```diff
@@ -1009,7 +1002,8 @@ int parseArguments(int argc, char* argv[], Arguments& args) {
 	int rc;
 	int c;
 	int idx;
-	while (1) {
+	bool report = false;
+	while (1 && !report) {
```

Comment:
In the [ticket](https://snowflakecomputing.atlassian.net/browse/FDBCORE-2177), you set the acceptance criteria as
`mako -m report <file1.json> <file2.json> ...`
My assumption was that mako would just merge all the reports and print the aggregated values. If this is correct, would it make sense to give other arguments?

Reply:
It doesn't but you could just ignore them right?

Reply:
We don't have a way to know ahead of time how many files we're aggregating in the mako report. So if I were to do something like ```mako -m report a.json b.json --procs 2```, how can I know where the list of files end? Current, I just read everything after the report and assume its a filename (I also have added logic in validateArguments in case it was an incorrect filename). Ex:
```
else if (strcmp(optarg, "report") == 0) {
		report = true;
		args.mode = MODE_REPORT;
		for (int i = optind; i < argc; i++) {
			args.report_files.push_back(argv[i]);
	}
}
```